### PR TITLE
Reorganizing navigation and document order

### PR DESCRIPTION
**Pull request by autodidaddict**
Reorganizes the left navigation and the flow of documents to be more ergonomic for the reader and to better facilitate progressive learning.

NOTE that this PR currently only contains the reorganization. There are a number of empty pages here that need to be filled out.

### CODE REVIEW

**Comment by andar1an** [comment]
I have found a few instances in preview where an "i" appears before a title. I am not sure if that is desired, but I will try to edit this comment with where I have seen them. I am not sure if it was done by this PR as when I searched for line in healthcheck under elixer for example it was not part of edited text. 

ElixirOTP/healthcheck.md
image

ElixirOTP/safeshutdown.md
image

**Comment by autodidaddict** [comment]
> I have found a few instances in preview where an "i" appears before a title. I am not sure if that is desired, but I will try to edit this comment with where I have seen them. I am not sure if it was done by this PR as when I searched for line in healthcheck under elixer for example it was not part of edited text.
> 
> ElixirOTP/healthcheck.md image
> 
> ElixirOTP/safeshutdown.md image

The markdown includes things like `:::info` and so on. These produce differently styled boxes. They're fine and get rendered nice and pretty in the preview.

File: docs/fundamentals/capabilities/create-provider/new-interface.md [hunk 1]
```diff
@@ -235,17 +235,15 @@ list PaymentMethods {
...
-
 If you modify this `.smithy` file, there are two useful `wash` commands to check it for errors:
 
 ```text
 wash lint
 wash validate
 ```
 
-Theese check the file against a predefined set of rules. The rules will be added to an improved over time.
+These check the file against a predefined set of rules. The rules will be added to an improved over time.
```
Comment (connorsmith256) [inline docs/fundamentals/capabilities/create-provider/new-interface.md]: Supernit: "added to and improved over time"

File: docs/fundamentals/wadm/_category_.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "label": "Managed Application Deployment",
```
Comment (connorsmith256) [inline docs/fundamentals/wadm/_category_.json]: Suggestion: I think `wadm` should be in the title here:
- "Managed Application Deployment (wadm)" or
- "wadm: Managed Application Deployment" or
- "Managed Application Deployment with wadm"

File: docs/fundamentals/wadm/usage.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Using wadm"
+title: "Using Wadm via Wash"
```
Comment (connorsmith256) [inline docs/fundamentals/wadm/usage.md:2]: Do we capitalize wadm and wash when using title case, or leave them lowercase?

File: docs/production/index.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Overview"
+date: 2020-01-19T00:00:00+00:00
+icon: "ti-map" # themify icon pack : https://themify.me/themify-icons
+description: "Overview of wasmCloud in Production"
+type: "docs"
+sidebar_position: 0
+---
+
+TBD
```
Comment (connorsmith256) [inline docs/production/index.md]: Ah, looks like this needs to be updated

**Review by connorsmith256 — CHANGES_REQUESTED** [review]
A couple nits/suggestions, and I found some remaining TBD pages:

- wasmCloud in Production -> Overview
- wasmCloud in Production -> Artifact Registries
- wasmCloud in Production -> Securing Applications -> Running a Policy Service
- Reference -> FAQ
- Reference -> Glossary

**Review by connorsmith256 — APPROVED** [review]

**Pull request merged**